spec/presenters/sufia/collection_presenter_spec.rb

```diff
@@ -58,4 +58,9 @@
     subject { presenter.total_items }
     it { is_expected.to eq 0 }
   end
+
+  describe "#identifier" do
+    subject { presenter.identifier }
```

Comment:
@awead Would adding shoulda-matchers help convey intention? The `{ is_expected.to be_nil }` is misleading for the specification of behavior; You are expecting delegation.